packages/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -1368,5 +1369,159 @@ describe('plugin-meetings', () => {
         );
       });
     });
+
+    describe('#isNeedHandleMainLocus', () => {
+      let meeting;
+      let newLocus;
+      beforeEach(() => {
+        meeting = {
+          controls: {},
+          self: {},
+        };
+        newLocus = {
+          controls: {},
+          self: {},
+        }
+      });
+      afterEach(() => {
+        sinon.restore();
+      });
+      it('check normal case will return true', () => {
+        sinon.stub(webex.meetings.meetingCollection, 'getActiveBreakoutLocus').returns(null);
+        LoggerProxy.logger.log = sinon.stub();
+        const result = webex.meetings.isNeedHandleMainLocus(meeting, newLocus);
+        assert.equal(result, true);
+        assert.calledWith(
+          LoggerProxy.logger.log,
+          'Meetings:index#isNeedHandleMainLocus --> this is a normal main session locusDTO update case'
+        );
+      });
+
+      it('check self joined and joined on this device, return true', () => {
+        sinon.stub(webex.meetings.meetingCollection, 'getActiveBreakoutLocus').returns(null);
+        newLocus.self.state = 'JOINED';
+        sinon.stub(MeetingsUtil, 'joinedOnThisDevice').returns(true);
+
+        LoggerProxy.logger.log = sinon.stub();
+        const result = webex.meetings.isNeedHandleMainLocus(meeting, newLocus);
+        assert.equal(result, true);
+        assert.calledWith(
+          LoggerProxy.logger.log,
+          'Meetings:index#isNeedHandleMainLocus --> self this device shown as JOINED in the main session'
+        );
+      });
+
+      it('check self joined and joined on this device, return false', () => {
```

Comment:
issue: I think this test description needs updating. It certainly doesn't describe what the test is doing.